collectors/ebpf.plugin/ebpf.h

```diff
@@ -108,6 +108,12 @@ typedef struct ebpf_tracepoint {
     char *event;
 } ebpf_tracepoint_t;
 
+enum ebpf_threads_status {
+    NETDATA_THREAD_EBPF_RUNNING,
+    NETDATA_THREAD_EBPF_STOPPING,
+    NETDATA_THREAD_EBPF_STOPPED
```

Comment:
Why do you need both `NETDATA_THREAD_EBPF_RUNNING` and `NETDATA_THREAD_EBPF_STOPPING` if you check only for `NETDATA_THREAD_EBPF_STOPPED`?

Reply:
The other two are not explicit tested, but they are present in codes like this:

```c
while (!ebpf_cachestat_exited) {
        usec_t dt = heartbeat_next(&hb, step);
        (void)dt;
        if (ebpf_cachestat_exited)
```

that could be rewritten for:

```c
while (ebpf_cachestat_exited == NETDATA_THREAD_EBPF_RUNNING) {
        usec_t dt = heartbeat_next(&hb, step);
        (void)dt;
        if (ebpf_cachestat_exited == NETDATA_THREAD_EBPF_STOPPING)
```

my understanding is that this rewrite is not necessary, but if you prefer I can update to do the usage explicit.

Reply:
I added them with latest commit.